docs-new/docs/graphql/core/auth/index.mdx

```diff
@@ -45,48 +45,25 @@ As you can see from this:
     defined by you. If the request/operation is allowed, it generates an
     SQL query, which includes the row/column-level constraints from the
     access control rules, and sends it to the database to perform the
-    required operation (*fetch the required rows for queries,
-    insert/edit rows for mutations, etc.*).
+    required operation (_fetch the required rows for queries,
+    insert/edit rows for mutations, etc._).
 
 **See more details about setting up authentication and access control
 at:**
 
-<div class="toctree" maxdepth="1">
+- [Authentication](/graphql/core/auth/authentication/index.mdx)
+- [Authorization / Access control](/graphql/core/auth/authorization/index.mdx)
 
-authentication/index authorization/index
-
-</div>
-
-<div class="admonition">
-
-Learn course
+:::info Learn course
 
 If you'd like to learn about authentication and authorization / access
 control by following a tutorial, check out our Learn course,
-[Authentication with
-Hasura](https://hasura.io/learn/graphql/hasura-auth-slack/introduction/).
-
-</div>
-
-<div class="admonition">
+[Authentication with Hasura](https://hasura.io/learn/graphql/hasura-auth-slack/introduction/).
 
-Additional Resources
+:::
 
-Enterprise Grade Authorization - [Watch
-Webinar](https://hasura.io/events/webinar/authorization-modeling-hasura/?pg=docs&plcmt=body&cta=watch-webinar&tech=).
+:::info Additional Resources
 
-</div>
+Enterprise Grade Authorization - [Watch Webinar](https://hasura.io/events/webinar/authorization-modeling-hasura/?pg=docs&plcmt=body&cta=watch-webinar&tech=).
 
-> <div class="admonition dhc">
->
-> Adding collaborators in Hasura Cloud
->
-> Hasura Cloud offers collaborators in addition to role management.
-> Collaborators can be granted access to specific portions of the
-> console, like Metrics. Additionally, adding collaborators allows you
-> to avoid sharing the <span
-> class="title-ref">HASURA-ADMIN-SECRET</span>. For more information
-> about collaborator management, see
-> `Collaborators in Hasura Cloud <manage_project_collaborators>`.
->
-> </div>
```

Comment:
This is a commented out in rst file. SO please add here as a comment instead of removing

```
<!--
.. hiding this as it mixes auth for the data plane with auth for the control plane and might be confusing

  .. admonition:: Adding collaborators in Hasura Cloud
    :class: dhc

    Hasura Cloud offers collaborators in addition to role management. Collaborators can be granted access to specific portions of the console, like Metrics. Additionally, adding collaborators allows you to avoid sharing the `HASURA-ADMIN-SECRET`. For more information about collaborator management, see :ref:`Collaborators in Hasura Cloud <manage_project_collaborators>`.
-->
```